packages/adapter-api/test/elements.test.ts

```diff
@@ -132,14 +203,26 @@ describe('Test elements.ts', () => {
       expect(isEqualElements(ot, otDiff)).toBeFalsy()
     })
 
+    it('should identify object types with different meta types', () => {
+      const otDiff1 = ot.clone()
+      const otDiff2 = ot.clone()
+
+      otDiff1.metaType = new TypeReference(new ElemID('test', 'meta1'))
+      otDiff2.metaType = undefined
+
+      expect(isEqualElements(ot, otDiff1)).toBeFalsy()
+      expect(isEqualElements(otDiff1, otDiff2)).toBeFalsy()
+      expect(isEqualElements(otDiff2, otDiff2)).toBeTruthy()
```

Comment:
supposed to compare to `ot`?
```suggestion
      expect(isEqualElements(ot, otDiff2)).toBeTruthy()
```

Reply:
Done.

Reply:
`ot` absolutely has a meta type.